### PR TITLE
Add artifact limit

### LINKED ISSUE #151: Reject >1 Artifact / Bounty
**Is your feature request related to a problem? Please describe.**
We need ability to scale, multi-artifact bounties present scaling challenge.

**Describe the solution you'd like**
Reject a configurable number of artifacts / bounty, refuse to post it to chain. Set this to 1 for launch.

### PR DESCRIPTION
**Pull request by Rizato**
Reads the field artifact_limit from consul or config file. 

Closes #151

### Related
* polyswarm/contractor#15
* polyswarm/infrastructure#110 
* polyswarm/orchestration#54

### CODE REVIEW

File: src/polyswarmd/config.py [hunk 1]
```diff
@@ -256,10 +257,11 @@ def from_config_file(cls, filename):
 
         commmunity = config.get('community')
         ipfs_uri = config.get('ipfs_uri')
+        artifact_limit = config.get('artifact_limit', 256)
```
Comment (mrtizmoatwork) [inline src/polyswarmd/config.py]: why default to 256 if you are changing stuff to limit it to 1? Maybe default to 1?

File: src/polyswarmd/config.py [hunk 2]
```diff
@@ -291,10 +293,11 @@ def from_consul(cls):
         config = json.loads(config.decode('utf-8'))
 
         ipfs_uri = config.get('ipfs_uri')
+        artifact_limit = config.get('artifact_limit', 256)
```
Comment (mrtizmoatwork) [inline src/polyswarmd/config.py]: also here, should probably default to 1??

File: src/polyswarmd/config.py (hunk 1 above)
Comment (Rizato) [reply src/polyswarmd/config.py]: The default should match the contract IMO. If we want less, we should explicitly set it.

Comment (mrtizmoatwork) [reply src/polyswarmd/config.py]: ok

**Pull request merged**